swift/internal/compiling.bzl

```diff
@@ -1567,10 +1578,15 @@ def _declare_compile_outputs(
         user_compile_flags = user_compile_flags,
     )
 
-    if not output_nature.emits_multiple_objects:
+    emit_bc = is_feature_enabled(
+        feature_configuration = feature_configuration,
+        feature_name = SWIFT_FEATURE_EMIT_BC,
+    )
+    if not output_nature.emits_multiple_objects or emit_bc:
         # If we're emitting a single object, we don't use an object map; we just
         # declare the output file that the compiler will generate and there are
         # no other partial outputs.
+        # If emit_bc is on, only one bitcode file is emitted
```

Comment:
why?

Reply:
tried `output-file-map`, it doesn't seem to work with `emic_bc`, `-###` revealed it just ignore the output option and default to `module-name.bc`

Reply:
even with `num-threads` set? curious.

Reply:
OK I'm missing something here
with `num-threads` on, swift will actually try to output multiple bc files, except that I can't find an option to control the output location, whatever client option I provide it just ignores it and output to the root
with single threaded wmo, it will behave like buck did, this seems intentional, I'm just gonna disable threading during bc_merge pipeline I think

Reply:
When you say root what do you mean?

Reply:
Interesting. Pretty fine with the single object file output if that makes things easier.